### PR TITLE
Add example of onboarding testgrid as a service from prow

### PR DESCRIPTION
**Pull request by chaodaiG**

### CODE REVIEW

File: standalone.md [hunk 1]
```diff
@@ -54,7 +54,31 @@ The TestGrid configuration located at `gs://your-bucket/config` can be rendered
...
+
+### Standalone Testgrid FE For Knative Prow
+
+#### Generate testgrid config
+
+Set up a prow job that generates testgrid config from prow configs with
+`configurator`. In knative scenario, the [configurator job example] writes the
+config to a GCS bucket `gs://knative-own-testgrid/config`.
+
+After this step, `testgrid.k8s.io/r/knative-own-testgrid` can already work, but
```
Comment (chases2) [inline standalone.md:67]: I would specify what you mean by "working": that you should see dashboards and dashboard tabs, but no test results.

File: standalone.md [hunk 2]
```diff
@@ -54,7 +54,31 @@ The TestGrid configuration located at `gs://your-bucket/config` can be rendered
...
+there is no data yet.
+
+#### Load data
+
+Create the [deployment] of `Updater` and `Summarizer` in testgrid cluster (Note:
+this can be any cluster of your choice), make sure that both deployments
```
Comment (chases2) [inline standalone.md:73]: I think this should be simply replaced with "your" or "a" kubernetes cluster. These are kubernetes deployments that can run anywhere, and for many users maybe shouldn't run in the k8s-community cluster.

File: standalone.md [hunk 3]
```diff
@@ -54,7 +54,31 @@ The TestGrid configuration located at `gs://your-bucket/config` can be rendered
...
+understand processing `gs://knative-own-testgrid`.
```
Comment (chases2) [inline standalone.md:74]: I would specify specifically that you pass them the flag to the file that Configurator generates, `--config=gs://knative-own-testgrid`. Less ambiguous than whether a deployment "understands processing" or not.

**Review by chases2 — COMMENTED** [review]
This looks great! I'm really excited about how great [testgrid.k8s.io/r/knative-own-testgrid] looks.

A few language nits, but this content is fantastic! I can take a second look at this PR tomorrow when I'm in the office.

File: standalone.md (hunk 1 above)
Comment (chaodaiG) [reply standalone.md]: done

File: standalone.md (hunk 2 above)
Comment (chaodaiG) [reply standalone.md]: done

File: standalone.md (hunk 3 above)
Comment (chaodaiG) [reply standalone.md]: done

**Review by chases2 — APPROVED** [review]

**Pull request merged**